### PR TITLE
feat(buttons): add new component - INNO-877

### PR DESCRIPTION
**Pull request by raduchiriac**

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *ema-styleguide* ready!

Built with commit 848d618f12a9c5c494d16879f2d74ddcbb815312

https://deploy-preview-12--ema-styleguide.netlify.com

File: framework/components/ema-buttons/ema-buttons.twig [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print result #}
+<button class="{{ _css_class }}"{{ _extra_attributes }}>{% block button_label_block _label %}</button>
```
Comment (emeryro) [inline framework/components/ema-buttons/ema-buttons.twig]: You don't even need to have this full twig file; just call ECL button here with `{% include '@ec-europa/ecl-buttons' %}`
(have a look at ema-links.twig).
Then in ema-buttons.config.js add extra_classes for each modifier (as it has been done in ema-links.config.js)

File: framework/base/variables/_colors.scss [hunk 2]
```diff
@@ -6,6 +6,7 @@ $ema-colors: (
     'blue-1': #13376d,
     'blue-2': #0b4ba0,
     'blue-3': #4172b3,
+    'blue-4': #0f3f82,
```
Comment (emeryro) [inline framework/base/variables/_colors.scss]: Where do this color comes from?

File: framework/components/ema-buttons/ema-buttons.config.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  title: 'Buttons',
+  label: 'Buttons',
+  preview: '@preview-website',
+  tags: ['atom'],
+  status: 'ready',
+  collated: true,
+  collator(markup, item) {
+    return `
+      <!-- Start: @${item.handle} -->\n
+      <h3 class="ecl-heading ecl-heading--h3">${item.label}\n</h3>\n
```
Comment (emeryro) [inline framework/components/ema-buttons/ema-buttons.config.js]: classes should be `ecl-heading ecl-heading--ema-h3`

File: framework/base/variables/_colors.scss (hunk 2 above)
Comment (raduchiriac) [reply framework/base/variables/_colors.scss]: I colorpicked from the PDF. Should we use `primary`?

**Pull request merged**